.vib/ruby/goss/vars.yaml

```diff
@@ -0,0 +1,18 @@
+binaries:
+  - bundle
+  - bundler
+  - gem
+  - rake
+  - ruby
+modules:
+  - bundler
+  - openssl
+  - readline
+directories:
+  - mode: "0755"
+    paths:
+      - /opt/bitnami/ruby
```

Comment:
There is no need to check this directory. There is no explicit mention in the Dockerfile or post-unpack.sh files.

Reply:
You may however need to check that the static library`lib/libruby-static.a` is included.